src/logo-system/social/index.md

```diff
@@ -74,6 +74,16 @@ Social end frames can be used at the end of animated or filmed content. They inc
     "/graphic-device/dot-use-examples/end-frame-1920-1080.webm"
 ] } %}
 
+#### Video description of video end frames
+
+This series of videos do not contain any audio.
+
+Social video end frames are shown above and below in different 3 different formats: 16:9 widescreen (above), 9:16 mobile view and 1:1 square (below).
```

Comment:
Do we need this sentence? Together, the headings might be clear enough that we're referring to the 3 videos